pkg/engine/utils.go

```diff
@@ -104,3 +104,15 @@ func ParseRegexPolicyResourceName(policyResourceName string) (string, bool) {
 	}
 	return strings.Trim(regex[1], " "), true
 }
+
+func GetAnchorsFromMap(anchorsMap map[string]interface{}) map[string]interface{} {
```

Comment:
Write test for this function

Reply:
It already has tests in validation_test.go